tools/ci/gha-install.py

```diff
@@ -4,50 +4,57 @@
 
 
 def install(omp=False, mpi=False, phdf5=False, dagmc=False, libmesh=False, ncrystal=False):
-    # Create build directory and change to it
-    shutil.rmtree('build', ignore_errors=True)
-    os.mkdir('build')
-    os.chdir('build')
-
-    # Build in debug mode by default with support for MCPL
-    cmake_cmd = ['cmake', '-DCMAKE_BUILD_TYPE=Debug', '-DOPENMC_USE_MCPL=on']
+    # List to store the CMake arguments
+    cmake_args = ['-DCMAKE_BUILD_TYPE=Debug', '-DOPENMC_USE_MCPL=on']
 
     # Turn off OpenMP if specified
     if not omp:
-        cmake_cmd.append('-DOPENMC_USE_OPENMP=off')
+        cmake_args.append('-DOPENMC_USE_OPENMP=off')
 
     # Use MPI wrappers when building in parallel
     if mpi:
-        cmake_cmd.append('-DOPENMC_USE_MPI=on')
+        cmake_args.append('-DOPENMC_USE_MPI=on')
 
     # Tell CMake to prefer parallel HDF5 if specified
     if phdf5:
         if not mpi:
-            raise ValueError('Parallel HDF5 must be used in '
-                             'conjunction with MPI.')
-        cmake_cmd.append('-DHDF5_PREFER_PARALLEL=ON')
+            raise ValueError('Parallel HDF5 must be used in conjunction with MPI.')
+        cmake_args.append('-DHDF5_PREFER_PARALLEL=ON')
     else:
-        cmake_cmd.append('-DHDF5_PREFER_PARALLEL=OFF')
+        cmake_args.append('-DHDF5_PREFER_PARALLEL=OFF')
 
     if dagmc:
-        cmake_cmd.append('-DOPENMC_USE_DAGMC=ON')
-        cmake_cmd.append('-DOPENMC_USE_UWUW=ON')
+        cmake_args.append('-DOPENMC_USE_DAGMC=ON')
+        cmake_args.append('-DOPENMC_USE_UWUW=ON')
         dagmc_path = os.environ.get('HOME') + '/DAGMC'
-        cmake_cmd.append('-DCMAKE_PREFIX_PATH=' + dagmc_path)
+        cmake_args.append('-DCMAKE_PREFIX_PATH=' + dagmc_path)
 
     if libmesh:
-        cmake_cmd.append('-DOPENMC_USE_LIBMESH=ON')
+        cmake_args.append('-DOPENMC_USE_LIBMESH=ON')
         libmesh_path = os.environ.get('HOME') + '/LIBMESH'
-        cmake_cmd.append('-DCMAKE_PREFIX_PATH=' + libmesh_path)
+        cmake_args.append('-DCMAKE_PREFIX_PATH=' + libmesh_path)
 
     if ncrystal:
-        cmake_cmd.append('-DOPENMC_USE_NCRYSTAL=ON')
+        cmake_args.append('-DOPENMC_USE_NCRYSTAL=ON')
 
     # Build in coverage mode for coverage testing
-    cmake_cmd.append('-DOPENMC_ENABLE_COVERAGE=on')
+    cmake_args.append('-DOPENMC_ENABLE_COVERAGE=on')
 
-    # Build and install
-    cmake_cmd.append('..')
+    # Set environment variable for SKBUILD
+    os.environ['SKBUILD_cmake_args'] = ';'.join(cmake_args)
+
+    # Run pip to build and install
+    pip_suffix = '--config-settings=cmake.args="' + ';'.join(cmake_args) + '"'
+    subprocess.check_call(['pip', '-v', 'install', '.[test,vtk,ci,ncrystal]', '--no-build-isolation', pip_suffix])
```

Comment:
Looks like we have ```--no-build-isolation``` running but we can remove it if that is the right thing to do?

```suggestion
    subprocess.check_call(['pip', '-v', 'install', '.[test,vtk,ci,ncrystal]', pip_suffix])
```